docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: '2.4'
```

Comment:
Docker deprecated depend_on ->service -> condition in v3. There is a replacement but that only works with swarms. 
https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on